### PR TITLE
Task-61-For-Testing

### PR DESCRIPTION
**Pull request by hnbmunoz**

### CODE REVIEW

**Comment by hnbmunoz** [comment]
styling for GR requires different approach had to use : https://github.com/react-icons/react-icons/issues/404?fbclid=IwAR2LIbl5Mrt1MknBZyQ3YlXevfCIYEqV6E3Uo-2rP1wLNEibsWeN4JPDfzk  as reference

File: react-client/src/Components/BookmarkPanel/AddBookmarkPanel.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { AiOutlineClose } from "react-icons/ai";
 
 const AddBookmarkPanel = ({ domType, elClassNames, saveBookmark, onClose, x, y, domId }) => (
   <div className='add__bookmark-panel' style={{ top: y + 270, left: x + 15 }}>
-    <span className='header'>
+    <span className='add__bookmark-header'>
```
Comment (reciosonny) [inline react-client/src/Components/BookmarkPanel/AddBookmarkPanel.js:7]: Incorrect BEM CSS class naming. Please refer here for proper naming convention of BEM: https://cssguidelin.es/#bem-like-naming

File: react-client/styles/components/_bookmarkpanel.scss [hunk 2]
```diff
@@ -1,6 +1,8 @@
 
 
 .bookmark-panel {
+    color: #263238 !important; 
+    font-family: Roboto monospace;
```
Comment (reciosonny) [inline react-client/styles/components/_bookmarkpanel.scss:5]: This is the one causing an issue. Please review

**Review by reciosonny — COMMENTED** [review]
Incorrect. Needs update.

**Pull request closed without merging**